tests/test_search.py

```diff
@@ -333,20 +335,20 @@ def get_search_results(self, es_query, url_params=None):
         search_obj = self.assertPostResponse(
```

Comment:
I feel you can combine this with:

```
while True:
    search_for_stuff(..)
    if search_obj.response.status_code == 200:
        break
```

Or am I completely off base?

Reply:
That's work. Thanks!

Reply:
```
url = self.build_url(url_params)
while True:
    search_response = search_for_stuff(url, ...)
    if search_obj.response.status_code == 200:
        break
    url = self.get_next_url(search_obj.response.headers)
```

Reply:
Will fix.